### PR TITLE
i made awesome code

### PR DESCRIPTION
**Pull request by stevenkussmaul**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,6 +16,8 @@
 </section>
 <a href="" id="anchor">this is an anchor</a>  -->
 
+<h1><p>THIS CODE IS AWESOME</p></h1>
```
Comment (schoeffa) [inline index.html:19]: AHHHHH!!! =O

Comment (karisP) [inline index.html:19]: it really is awesome!!

Comment (karisP) [inline index.html:19]: this is great steven!

**Review by karisP — COMMENTED** [review]
hey hey cool

**Pull request merged**